cli/tests/conftest.py

```diff
@@ -22,6 +22,8 @@
 from sqlalchemy import Engine, create_engine
 from sqlalchemy.orm import Session, sessionmaker
 
+os.environ["PROJECT_CONFIGS_DIR"] = str(pathlib.Path(__file__).parents[2] / "projects/configs")
```

Comment:
I know I wrote this, but what about making the configs folder a python module and using pip and importlib to access it instead of this shaky parents[2] structure? Then we don't need to install editable either

Reply:
Sounds good, but maybe make a separate issue and PR for it? Because this PR is getting quite bloated already as it is.

Reply:
created #326